packages/typespec-ts/src/transform/transform.ts

```diff
@@ -60,7 +60,6 @@ export async function transformRLCModel(
   const importSet = new Map<ImportKind, Set<string>>();
   const paths: Paths = transformPaths(program, client, dpgContext);
   const schemas: Schema[] = transformSchemas(program, client, dpgContext);
-
```

Comment:
We don't need a transform part for samples in the emitter ?

Reply:
updated